README.md

```diff
@@ -26,97 +25,93 @@ import JSONRPC from 'jsonrpc-dispatch'
 ```
 ## Using JSONRPC
 
-Create a JSONRPC instance and pass in an object with function handlers for all supported requests and notifications.
+The JSONRPC takes 2 arguments, a dispatcher and an object mapping of
+methods it will take requests for.
 
-*Requests* are always passed an additional parameter ``next`` which takes 2 arguments (error, result).
 
 ```js
-const jsonrpc = new JSONRPC({
-  // Example request which adds x and y
-  add: function(x,y, next) {
-    next(null, x+y)
-  },
-
-  // Example request which divides x by y
-  divide: function(x,y, next) {
-    if (y==0) {
-      next('cannot divide by zero')
-      return
-    }
-
-    next(null, x/y)
-  },
-
-  // Example notification
-  hello: function() {
-    console.log('a notification occurred')
+const jsonrpc = new JSONRPC([dispatcher], [methods]);
+```
+
+| name | type |  description |
```

Comment:
I'd like the table to be readable from the markdown.  Checkout http://www.tablesgenerator.com/markdown_tables  (it can import your existing markup)